docs/psoc6/quickref.rst

```diff
@@ -425,7 +435,7 @@ SPI object is created with default settings or settings of previous initializati
 ::
     
     from machine import SPI
-    spi = SPI(0) # Default assignment: id=0, sck=P9_2  ,MOSI=P9_0, MISO=P9_1, baudrate=1000000, Polarity=0, Phase=0, bits=8, firstbit=SPI.MSB
+    spi = SPI(0, sck='P11_2', mosi='P11_0', miso='P11_1') # Default assignment: id=0, sck=P9_2  ,MOSI=P9_0, MISO=P9_1
```

Comment:
Here pins and comment is mismatched.  I think it should be 
spi = SPI(0, sck='P9_2', mosi='P9_0', miso='P9_1')   # Default assignment: id=0, baudrate=1000000, Polarity=0, Phase=0, bits=8, firstbit=SPI.MSB

Reply:
Yup. Let me fix it!